src/netbase.cpp

```diff
@@ -682,10 +682,11 @@ bool ConnectThroughProxy(const Proxy& proxy, const std::string& strDest, uint16_
     return true;
 }
 
-bool LookupSubNet(const std::string& subnet_str, CSubNet& subnet_out)
+CSubNet LookupSubNet(const std::string& subnet_str)
 {
+    CSubNet subnet;
     if (!ContainsNoNUL(subnet_str)) {
-        return false;
+        return subnet;
```

Comment:
nit: maybe add `assert(!subnet.IsValid());` after `CSubNet subnet;` to make it obvious that we return an invalid object and to guard against future changes that construct a valid `CSubNet` by default.

Reply:
Make sense, done!